Sources/TuistCore/ValueGraph/ValueGraphTraverser.swift

```diff
@@ -262,7 +262,13 @@ public class ValueGraphTraverser: GraphTraversing {
 
         // AppClip dependencies
         if target.target.isAppClip {
-            let path = try SDKNode.appClip(status: .required).path
+            let path = try SystemFrameworkMetadataProvider().loadMetadata(
```

Comment:
Shall we make the metadata provider a property in this class?

Reply:
makes sense, let me change that 👍